requirements.txt

```diff
@@ -18,3 +18,6 @@ flake8==3.9.0
 black==22.3.0
 elastic-apm==6.1.3
 django-extensions==3.1.1
+schema-salad==4.5.20191229160203
+cwltool==1.0.20190906054215
```

Comment:
cwltool and schema-salad would be installed here: https://github.com/mskcc/ridgeback/blob/fix/requirements/requirements-toil.txt#L5, since they are dependencies of toil. I think you might not need to specify it here

Reply:
I will move it around if this other test completes

Reply:
I adjusted the dependencies and built the singularity image locally with no issues. Also, it now passes the travis test.

That said, we should talk about this since I made more changes than expected.